src/app_service/service/community/service.nim

```diff
@@ -446,6 +451,12 @@ QtObject:
 
             let data = CommunityChatArgs(chat: updatedChat)
             self.events.emit(SIGNAL_COMMUNITY_CHANNEL_EDITED, data)
+
+      # members list was changed
+      if community.isMember and community.members != prev_community.members:
+        self.events.emit(SIGNAL_COMMUNITY_MEMBERS_CHANGED, 
+        CommunityMembersArgs(communityId: community.id, members: community.members))
```

Comment:
FYI, when we put a line on two lines, we add an additional tabulation (2 spaces) on the new lines, so differentiate actual new lines of code vs the previous one being continued.
Not a big, deal just letting you know.